src/appcast.cpp

```diff
@@ -126,7 +130,23 @@ void XMLCALL OnStartElement(void *data, const char *name, const char **attrs)
     {
         if ( strcmp(name, NODE_RELNOTES) == 0 )
         {
-            ctxt.in_relnotes++;
```

Comment:
This (and the corresponding change when closing the tag) doesn’t seem right. `in_relnotes` keeps track of the position in the document, sometimes pretending the tag wasn’t there, as you now do, seems dangerous. Why not keep track of the name instead?

Reply:
needs to skip unsuitable node. it was quick idea for that. and, I agree, not good idea. I modify it. I don't understand about the name. what the name?

Reply:
xml looks like
<sparkle:releaseNotesLink>http://download.somesite.com/changes/changes.html</sparkle:releaseNotesLink>
<sparkle:releaseNotesLink xml:lang="de-DE">http://download.somesite.com/changes/changes_de.html</sparkle:releaseNotesLink>